browser-test/src/support/index.ts

```diff
@@ -205,3 +212,19 @@ export const validateAccessibility = async (page: Page) => {
 
   expect(results).toHaveNoA11yViolations()
 }
+
+export const validateScreenshot = async (
```

Comment:
Let's add a function comment explaining that it takes a screenshot and saves it to a file `__snapshots__/test_file_name/name-of-the-test.png`. And that depending on whether screenshot is already present it is either diffing or adding a new screenshot. I think this function will be the main place where engineers learn about how screenshots work so it's important to have a good explanation here.